### ISSUE TITLE
Add documentation for all configuration options.

### ISSUE THREAD

**Issue by damonkohler** [googlecartographer/cartographer#54]

**Comment by SirVer** [comment]
Yes, I did: http://bazaar.launchpad.net/~widelands-dev/widelands/trunk/view/head:/doc/sphinx/extract_rst.py

That is a simple script that looks for `/* RST ` in source files and extracts all lines till the end of the comment, remove the common prefix and dump it into a .rst file. Not very elegant, but functional.
